### DISCUSSION TITLE
How to take multiple initiative values for same actor?

### DISCUSSION THREAD

**Discussion by itsmegrave** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#4] [Q&A] [answered]
Hey dudes!
Some PCs and monsters can have multiple attacks. There's a way to emulate this on the game system?

Ty

**Comment by aMediocreDad** [comment]
If you are thinking about multiple initiative values. Then no AFAIK there is no way to support this within Foundry yet.

**Reply by itsmegrave** [reply]
thanks for the help mate!
Great job with the game system :)

**Comment by pwatson100** [comment, marked as answer]
have a look at the Reverse Initiative module.  it does reverse the initiative scores but also allow you to add an actor more than once hence rolling initiative for them twice.  We have been using it for Alien and it works really well.
